### DISCUSSION TITLE
Native wifi card compatible or not ?

### DISCUSSION THREAD

**Discussion by sinclairfr** [valnoxy/t480-oc#32] [Q&A] [answered]
Hello,
Thanks a lot for this github repo.
I was wondering : does the native wifi card work?
If not, what's a viable option for a replacement ?
Thanks

**Comment by valnoxy** [comment, marked as answer]
Hey! Yes, the native Intel wifi card will work but without Handoff & AirDrop.

**Reply by trevorwdunn** [reply]
I had a lot of trouble with it in the recovery environment. Had to start with an old version (Monterey), then upgrade the EFI and update to Ventura from there. Once Monterey was installed the wifi kexts from both configurations worked. Prior to installing Monterey, only the one from early 2022 was working for me.

**Comment by Mifaxav** [comment]
@valnoxy i have a DW1560 but cant get anything to work on macos ventura. (Except wifi, bt and airdrop) Any ides why this might not be working? Should i downgrade to big sur for these features?
